src/DssSpell.sol

```diff
@@ -16,25 +16,39 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-// pragma experimental ABIEncoderV2;
+pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
+interface StarknetBridgeLike {
+    function close() external;
+    function isOpen() external returns (uint256);
+}
+
+interface StarknetGovRelayLike {
+    function relay(uint256 spell) external;
+}
+
+interface StarknetEscrowLike {
+    function approve(address token, address spender, uint256 value) external;
+}
+
+
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/57700b7c1eab15c213e5c92cd9ebf5de9df44f24/governance/votes/Executive%20vote%20-%20October%2019%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/27dc14fd37c03c152eca32296076f5603c9fd4db/governance/votes/Executive%20vote%20-%20October%2026%2C%202022.md -q -O - 2>/dev/null)"
 
     string public constant override description =
-        "2022-10-19 MakerDAO Executive Spell | Hash: 0x300ef27d3eee7338f3619ea697a6e9f1c85e14f6547342c8f4a40d05f26ccd1f";
+        "2022-10-26 MakerDAO Executive Spell | Hash: 0x2e8fa79dc9702f6d3b8b03523fc45c4f3f95751a833e52958d64182b0ec8b2a5";
 
-    // Turn office hours off
-    function officeHours() public override returns (bool) {
+    // Turn office hours on
+    /* function officeHours() public override returns (bool) {
```

Comment:
I think we tend to use the single line comments so this would be `//` on each line